source/guide/3.2/plugin-vsphere.md

```diff
@@ -178,15 +204,29 @@ properties:
 
 Node by node explanation:
 
-1. Creates a server. In the server 'networking' property we spefied desired domain name as 'example.com', additional DNS server 8.8.8.8, and three existing networks we want to connect to: example_management_network, example_external_network and example_other_network. In the 'server' property we specified server name as example_server, vm template name as example_server_template, number of cpus as 1, and RAM as 512 MB.
+1. Creates a server from the example_server_template. This server will have 1 CPU and 512MB of RAM and be attached to the three networks (example_management_network, example_external_network, and example_other_network). The IP on the example_external_network will be statically assigned. The DNS server 192.0.2.1 will be assigned as a DNS server.
 
-2. Creates a network. We specified network name as example_network, network vLAN id as 1, and an existing vSwitch name we want to connect to as example_vswitch.
+2. Creates a network. We specified the network name as example_network, with network VLAN ID as 101, and an existing vswitch name we want to connect to as example_vswitch. This results in us creating a port group called 'example_network' on the 'example_vswitch' vswitch, tagged with VLAN ID 101.
 
-3. Creates a storage. We specified desired storage size as 1 GB and wish to add this storage to example_server vm.
+3. Creates a new virtual hard disk (storage). We specified desired storage size as 1 GB and wish to add this storage to example_server vm.
 
 {% endgcloak %}
 
 
+# Misc
+
+## Virtual machine template
+Template should have:
+
+* root disk with OS and vSphere Tools installed.
+* Linux servers should have SSH server installed and a user account, with manager and agent SSH keys in authorized_hosts.
+* Windows servers should be [configured to support winrm scripting.](plugin-windows-agent-installer.html)
+
```

Comment:
There should also be a reference to having NOPASSWD for sudo group and setting '%sudo   ALL=(ALL:ALL) ALL' to '%sudo   ALL=(ALL:ALL) NOPASSWD:ALL'